nixos/lib/test-driver/src/test_driver/errors.py

```diff
@@ -0,0 +1,13 @@
+from unittest import TestCase
+
```

Comment:
can you please add comments/docstrings to describe what these errors semantically mean? I don't understand what a NixOS error shall be

Reply:
Done.
And I agree, `NixOSError` isn't too descriptive, changed :)